README.md

```diff
@@ -1,40 +1,32 @@
-# Eclipse Public Access Submission System (PASS)
+# Eclipse Public Access Submission System (PASS) ![Pass Main](https://github.com/eclipse-pass/main/actions/workflows/pass-complete-release.yml/badge.svg)
 
-## Introduction
+The Public Access Submission System (PASS) is a comprehensive platform designed to help researchers comply with the 
+access policies mandated by their funders and institutions.
 
-The Public Access Submission System (PASS) is a platform to assist researchers in complying with the access policies of their funders and institutions.
+Navigating the complex landscape of public access requirements can be challenging, as policies vary significantly across
+federal agencies, each with its own unique compliance process. These differing requirements often place an
+administrative burden on researchers and institutions, particularly when research is funded by multiple agencies, making
+compliance a complex and time-consuming task.
 
-While many federal agencies have policies that require research results to be made publicly accessible, the requirements to comply with these vary greatly from one agency to another. These heterogeneous processes for compliance have become burdensome for researchers and their institutions. For research that is funded by multiple agencies, it can be especially complicated to ensure compliance with each funder’s access policy.
+In addition, many universities have adopted their own open access policies, which further mandate that research outputs
+be publicly accessible. This overlap means that a single publication can fall under multiple policies, each with
+distinct workflows and submission criteria. PASS simplifies this process by consolidating and streamlining the
+submission workflows, enabling researchers to efficiently meet the diverse compliance requirements.
 
-Meanwhile, many universities have implemented an open access policy, which also requires research outputs to be made publicly accessible. As a result, each publication can be subject to multiple policies each with its own workflow and requirements.
+## Documentation
 
-## Purpose
+* [Eclipse PASS Website](https://eclipse-pass.org/index.html)
+* [PASS Documentation](https://docs.eclipse-pass.org/)
+* [PASS JHU User Guide](https://pass.jhu.edu/guide/)
 
-The PASS platform offers a unified approach for compliance, allowing researchers to fulfil the access policies of their institutions and funders through a single website.
+## Repositories
 
-PASS aims to:
+* [PASS Core](https://github.com/eclipse-pass/pass-core)
+* [PASS UI](https://github.com/eclipse-pass/pass-ui)
+* [PASS Support](https://github.com/eclipse-pass/pass-support)
+* [PASS Acceptance Testing](https://github.com/eclipse-pass)
+* [PASS Docker](https://github.com/eclipse-pass/pass-docker)
```

Comment:
Should we include the PASS Documentation repo too?

Reply:
Done.